plugins/handlers/gallery.py

```diff
@@ -285,17 +359,26 @@ def safe_resize(source, destination, size):
 
     This makes use of `RESIZE_METHODS` to determine which resize handler to use.
     """
-    resize_method = evaluate_tests(RESIZE_METHODS, os.path.basename(source))
+    resize_method = evaluate_tests(RESIZE_METHODS, {"basename": os.path.basename(source)})
     resize_method(source,
                   destination,
                   f"{size.width}x{size.height}")
 
 
+# TODO: Remove this.
```

Comment:
For sure. It's confusing just how many resize wrappers there are. Ideally this wouldn't be necessary.